docs/theming.md

```diff
@@ -69,6 +72,12 @@ Here's the markup:
       .element-card > .mdc-card__media {
         height: 9em;
       }
+      #demo-absolute-fab {
```

Comment:
Nit: Add an empty line before this selector

Reply:
Thanks for catching it!